docs/tutorials/namespace-tutorial.md

```diff
@@ -56,7 +67,10 @@ Name: newnamespace
 | Configurations |                               |
 ```
 
-By default, `Epinio` creates the namespace `workspace` during installation. If no previous local state existed the `epinio login` command used to get access to the installation also set this namespace as the current namespace. To see this use:
+Epinio creates the default namespace `workspace` at installation.
```

Comment:
```suggestion
Epinio creates the default namespace `workspace` at installation time.
```
Maybe?

Reply:
I think it's clear that the 'time' is implied from the context. It's clear that a place is not involved as in:

God creates the garden at Eden.

But sure, either is fine.